KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -14,462 +14,446 @@ public enum UITexts {
-        public static let signupPendingMessage = "Text.User.SignupPendingMessage".localized
-        public static let forgottenPassword = "Text.User.ForgottenPassword".localized
+        public static let firstName = "kh_uisdk_first_name_hint".localized
+        public static let lastName = "kh_uisdk_last_name_hint".localized
+        public static let email = "kh_uisdk_email_hint".localized
+        public static let mobilePhone = "kh_uisdk_mobile_phone_number_hint".localized
+        public static let signupEmailSubscribeCopy = "kh_uisdk_signupEmailSubscribeCopy".localized
+        public static let signupPendingMessage = "kh_uisdk_signupPendingMessage".localized
+        public static let forgottenPassword = "kh_uisdk_forgottenPassword".localized
+        public static let commentOptional = "kh_uisdk_additional_comments".localized
     }
 
     /* Errors */
     public enum Errors {
-        public static let flightNumberValidatorError = "Text.Error.FlightNumberValidationHint".localized
-        public static let noDetailsAvailable = "Text.Error.NoDetails".localized
-        public static let somethingWentWrong = "Text.Error.SomethingWentWrong".localized
-        public static let cantSendEmail = "Text.Error.Alert.CantSendEmail".localized
-        public static let getUserFail = "Text.Error.GetUserFails".localized
-        public static let forceUpdateTitle = "Text.Error.Alert.ForceUpdateTitle".localized
-        public static let forceUpdateMessage = "Text.Error.Alert.ForceUpdateMessage".localized
-        public static let forceUpdateButton = "Text.Error.Alert.ForceUpdateButton".localized
-        public static let prebookingWithinTheHour = "Text.Error.PrebookingWithinTheHour".localized
-        public static let missingPaymentSDKToken = "Text.Errors.failedToInitialisePaymentSetup".localized
-        public static let noResultsFound = "Text.Errors.NoResultsFound".localized
-        public static let missingPhoneNumber = "Text.Errors.MissingPhoneNumber".localized
-        public static let invalidPhoneNumber = "Text.Errors.InvalidPhoneNumber".localized
-        public static let insufficientBalanceForLoyaltyBurning = "Text.Errors.InsufficientBalanceForLoyaltyBurning".localized
-        public static let unsupportedCurrency = "Text.Error.UnsupportedCurrency".localized
-        public static let unknownLoyaltyError = "Text.Error.UnkownLoyaltyError".localized
-        public static let loyaltyModeNotEligibleForPreAuth = "Text.Error.LoyaltyModeNotEligibleForPreAuth".localized
+        public static let flightNumberValidatorError = "kh_uisdk_errors_flightNumberValidatorError".localized
+        public static let noDetailsAvailable = "kh_uisdk_something_went_wrong".localized
+        public static let somethingWentWrong = "kh_uisdk_something_went_wrong".localized
+        public static let cantSendEmail = "kh_uisdk_errors_cantSendEmail".localized
+        public static let getUserFail = "kh_uisdk_errors_getUserFail".localized
```

Comment:
key does not conform to standard

Reply:
not going to add this comment to all strings as they are too many. just please check all and make sure they conform to the agreed upon standard